spec/system/articles/user_visits_an_article_spec.rb

```diff
@@ -171,7 +172,7 @@
       it "does not the article edit link" do
```

Comment:
Quick, non-blocking suggestion similar to above!:
```suggestion
      it "does not render the article edit link" do
```